docs/integrations/sources/s3.md

```diff
@@ -152,66 +172,105 @@ Please note, the S3 Source connector used to infer schemas from all the availabl
-  - Together with `path_pattern`, there are multiple ways to specify the files to sync. For example, all the following configs are equivalent:
-    - `path_prefix` = `<empty>`, `path_pattern` = `path1/path2/myFolder/**/*`.
-    - `path_prefix` = `path1/`, `path_pattern` = `path2/myFolder/**/*.csv`.
-    - `path_prefix` = `path1/path2/` and `path_pattern` = `myFolder/**/*.csv`
-    - `path_prefix` = `path1/path2/myFolder/`, `path_pattern` = `**/*.csv`. This is the most efficient one because the directories are filtered earlier in the S3 API call. However, the difference in efficiency is usually negligible.
-  - The rationale of having both `path_prefix` and `path_pattern` is to accommodate as many use cases as possible. If you found them confusing, feel free to ignore `path_prefix` and just set the `path_pattern`.
-- `endpoint` : optional parameter that allow using of non Amazon S3 compatible services. Leave it blank for using default Amazon serivce.
-- `use_ssl` : Allows using custom servers that configured to use plain http. Ignored in case of using Amazon service.
-- `verify_ssl_cert` : Skip ssl validity check in case of using custom servers with self signed certificates. Ignored in case of using Amazon service.
+- **AWS Access Key ID**: One half of the [required credentials](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) for accessing a private bucket.
+- **AWS Secret Access Key**: The other half of the [required credentials](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) for accessing a private bucket.
+- **Path Prefix**: An optional string that limits the files returned by AWS when listing files to only those starting with the specified prefix. This is different than the **Path Pattern**, as the prefix is applied directly to the API call made to S3, rather than being filtered within Airbyte. It does not accept pattern-style symbols like wildcards (`*`). We recommend using this filter if your bucket has many folders and files that are unrelated to this stream, and all the relevant files will always reside under the specified prefix.
+  - Together with the **Path Pattern**, there are multiple ways to specify the files to sync. For example, all the following configurations are equivalent:
+    - **Prefix** = `<empty>`, **Pattern** = `path1/path2/myFolder/**/*`
+    - **Prefix** = `path1/`, **Pattern** = `path2/myFolder/**/*.csv`
+    - **Prefix** = `path1/path2/`, **Pattern** = `myFolder/**/*.csv`
+    - **Prefix** = `path1/path2/myFolder/`, **Pattern** = `**/*.csv`.
+
+  :::note
+  The last combination is the most efficient one, because the directories are filtered earlier in the S3 API call. However, the difference in efficiency is usually negligible.
```

Comment:
`However, the difference in efficiency is usually negligible.` This statement is confusing. So as a user, should I do this / is it impactful, or not? 😅 If it's not impactful, why is it here at all? 

if you're not sure I would just remove that statement

Reply:
Makes sense, it is kind of self-defeating haha :) Removed